modules/raft/src/main/java/org/apache/ignite/raft/jraft/core/FSMCallerImpl.java

```diff
@@ -665,6 +676,27 @@ private void doSnapshotLoad(final LoadSnapshotClosure done) {
             setError(e);
             return;
         }
+
+        // JRaft tests use such strange metas, so we have to protect... in production, these are never null.
+        if (meta.peersList() != null && meta.learnersList() != null) {
+            ConfigurationEntry configurationEntry = new ConfigurationEntry(
+                    snapshotId.copy(),
+                    new Configuration(
+                            meta.peersList().stream().map(PeerId::parsePeer).collect(toList()),
+                            meta.learnersList().stream().map(PeerId::parsePeer).collect(toList())
+                    ),
+                    new Configuration()
+            );
+            if (meta.oldPeersList() != null && !meta.oldPeersList().isEmpty()) {
+                configurationEntry.setOldConf(new Configuration(
+                        meta.oldPeersList().stream().map(PeerId::parsePeer).collect(toList()),
```

Comment:
Are you asking about https://github.com/apache/ignite-3/commit/aacb6ab15651a49f08a3841760537823cb8c34cf ? If yes, then yes, this commit was merged to `main` before I branched from `main`, my branch contains it.

Am I doing something wrong? I can see in my tests that peers are now identified with strings like `itrst_t_0` instead of `127.0.1.1:3344`, but everything seems working fine.

Reply:
No, nothing's wrong here, I'm worrying about conflicts, that's it